geomstats/learning/mdm.py

```diff
@@ -11,26 +11,25 @@
 
 
 class RiemannianMinimumDistanceToMeanClassifier:
-    r"""Minimum Distance to Mean (MDM) classifier on manifolds.
+    """Minimum Distance to Mean (MDM) classifier on manifolds.
 
     Classification by nearest centroid. For each of the given classes, a
     centroid is estimated according to the chosen metric. Then, for each new
-    point, the class is affected according to the nearest centroid (see
-    [BBCJ2012]_).
+    point, the class is affected according to the nearest centroid [BBCJ2012]_.
 
     Parameters
     ----------
     riemannian_metric : RiemannianMetric
         Riemannian metric to be used.
-    n_classes : int
-        Number of classes.
 
     Attributes
     ----------
-    mean_estimates_ : list
-        If fit, centroids computed on training set.
+    n_classes_ : int
+        If fit, number of classes.
```

Comment:
Do you think this attribute is necessary? I would simply remove it as we can simply compute this variable from `len(estimator.mean_estimates_)` or, simpler, `len(estimator.classes_)`

Reply:
You are right: we can remove this parameter.